### PR TITLE
Redesign pages

### PR DESCRIPTION
**Pull request by sbfnk**
Redesign to:
- use `leaflet` for maps
- split information more into multiple pages, to reduce the amount of information on each page
- some minor CSS tweaks to improve readability

Introduces some additional dependencies. From what I can see:
- `xaringanExtra`
- `leaflet`
- `rgdal`

### CODE REVIEW

File: bin/update_website.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@ Rscript utils/update_paper.R
 ## Manually force a UK page update
 Rscript -e 'rmarkdown::render("_posts/national/united-kingdom/united-kingdom.Rmd")'
 
-## Update all posts
-Rscript utils/update_posts.R
+## Update all Rscript
+posts utils/update_posts.R
```
Comment (seabbs) [inline bin/update_website.sh:20]: ?

Comment (sbfnk) [reply bin/update_website.sh]: fixed

**Review by seabbs — APPROVED** [review]
Phew quite a lot of heavy lifting in there. Just checking out locally to test but looks good.

**Comment by seabbs** [comment]
👯‍♂️ looks good to me. Merging and will fix if issues crop up.

**Pull request merged**